### PR TITLE
Fix DVM and DIM results from discovery service

### PR DESCRIPTION
**Pull request by djwhatle**
Resolves 
image

image

The link generated on the direct image migration was incorrect and didn't include the name of the resource.

This also renames the two resources in the discovery view so that they have the same name as their CR counterpart to reduce confusion for anyone attempting to `oc get` the resources.

### CODE REVIEW

**Comment by djwhatle** [comment]
Before: 

```                                                                                                                                                                      
          "kind": "DirectImage",                                                                                                                                                                                      
          "namespace": "openshift-migration",                                                                                                                                                                         
          "name": "f73abf40-9230-11eb-8b87-d7a540cbec50-zrqtk",                                                                                                                                                       
          "objectLink": "/namespaces/openshift-migration/directimages/:directimage"                                                                                                                                                                      
```

After
```
          "kind": "DirectImageMigration",
          "namespace": "openshift-migration",
          "name": "f73abf40-9230-11eb-8b87-d7a540cbec50-zrqtk",
          "objectLink": "/namespaces/openshift-migration/directimages/f73abf40-9230-11eb-8b87-d7a540cbec50-zrqtk"
```

File: pkg/controller/discovery/web/direct.go [hunk 1]
```diff
@@ -292,12 +292,12 @@ func (h DirectVolumeHandler) Link(m *model.DirectVolume) string {
...
 // Build self link.
-func (h DirectImageHandler) Link(m *model.DirectImage) string {
+func (h DirectImageMigrationHandler) Link(m *model.DirectImageMigration) string {
 	return h.BaseHandler.Link(
 		DirectImageRoot,
 		Params{
-			NsParam:           m.Namespace,
-			DirectVolumeParam: m.Name,
+			NsParam:          m.Namespace,
+			DirectImageParam: m.Name,
```
Comment (pranavgaikwad) [inline pkg/controller/discovery/web/direct.go:300]: Did this cause the link to break?

**Review by pranavgaikwad — APPROVED** [review]
VISIACK

File: pkg/controller/discovery/web/direct.go (hunk 1 above)
Comment (djwhatle) [reply pkg/controller/discovery/web/direct.go:300]: Yeah I think that was it

Comment (djwhatle) [reply pkg/controller/discovery/web/direct.go:300]: And the handler was missing.

**Pull request merged**